src/qt/window.cpp

```diff
@@ -1128,7 +1128,7 @@ wxSize wxWindowQt::DoGetBestSize() const
 {
     const wxSize size = wxWindowBase::DoGetBestSize();
 
-    if ( dynamic_cast<wxQtWidget*>(GetHandle()) )
+    if ( !GetHandle() || dynamic_cast<wxQtWidget*>(GetHandle()) )
```

Comment:
Is this change just an optimization? Because AFAICS if it's null, dynamic cast would fail anyhow.

This being said, I think it would be best to have some `QtIsNative()` virtual function instead of using this dynamic cast in the first place (if I understand correctly what it is doing).